hack/test-cmd.sh

```diff
@@ -67,6 +68,7 @@ export ETCD_PEER_PORT=${ETCD_PEER_PORT:-27001}
 setup_env_vars
 export SUDO=''
 mkdir -p "${ETCD_DATA_DIR}" "${VOLUME_DIR}" "${FAKE_HOME_DIR}" "${MASTER_CONFIG_DIR}" "${NODE_CONFIG_DIR}" "${LOG_DIR}"
+reset_tmp_dir
```

Comment:
Ok. This makes sense. We have `setup_env_vars` for the server in extended tests, can we have something similar for test environment variables? That would make it much easier to standardize `TEMPDIR` for instance.